desktop/desktop/lib/cookie-auth/index.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
```

Comment:
Like other `lib` modules, it seems we could move this one to the main `client/lib`.